### PR TITLE
Adding Bengali Language to Docs

### PR DESCRIPTION
**Pull request by abdulmominsakib**
I tried to cover all of the stuff in the docs. If there is any left you can mention it. @rrousselGit

### CODE REVIEW

**Comment by rrousselGit** [comment]
Hello!

The CI complains about broken links. Could you fix that? Thanks :pray:

**Comment by abdulmominsakib** [comment]
Broken links fixed ✅

File: website/tsconfig.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
   "extends": "@tsconfig/docusaurus/tsconfig.json",
-  "include": ["src/"]
-}
+  "include": [
+    "src/"
+  ]
+}
```
Comment (rrousselGit) [inline website/tsconfig.json:6]:
```suggestion
  "include": ["src/"]
}
```

**Comment by rrousselGit** [comment]
Thanks!

**Pull request merged**